Polygon.js

```diff
@@ -0,0 +1,129 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?://(?:www\\.)polygon\\.com",
```

Comment:
Any reason you have the `www` in parenthesis here? Is it optional (then you would need to add a question mark afterwards)? Otherwise I guess you can delete the parenthesis around the `www`.